src/insights/competition.ts

```diff
@@ -71,3 +86,13 @@ export interface QueryGetPNLCompetitionLeaderboardRequest {
   competitionId: string;
   market?: string;
 }
+
+export interface QueryGetLotteryCompetitionLeaderboardResponse {
+  entries: LotteryLeaderboardEntry[];
+  meta: TimeMeta;
```

Comment:
Do double-check that the insights API query function returns a meta field
https://test-api-insights.carbon.network/competition/leaderboardlottery?competitionId=tradingcomp7